test/libsolidity/syntaxTests/controlFlow/mappingReturn/named_err.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    function f() internal pure returns (mapping(uint=>uint) storage r) { }
+}
+// ----
+// TypeError: (53-82): This variable is of mapping type and might be returned without assignment and could be used uninitialized. Assign the variable (potentially from itself) to fix this error.
```

Comment:
Since the actual control flow analysis is extensively tested for storage returns, I would say that it's sufficient to only check that mappings are considered at all, instead of duplicating all tests in ``syntaxTests/controlFlow/storageReturn``.